css/style.css

```diff
@@ -217,35 +226,119 @@ q::after {
     margin: 0;
     padding: 100px 50px 50px;
   }
-}
 
-/**
- ********************
- *
- *  height: auto 
-   no height, adaptable 
+  /*  ____________  store page and form   ___________________ */
 
+  .form_wrapper {
+    width: 100%;
+    display: flex;
+  }
 
-   max height 
+  .form {
+    width: 50%;
+  }
 
+  .form_poster {
+    width: 50%;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
 
+  .form_field {
+    width: 75%;
+    margin: 20% auto;
+    text-align: left;
+    border: 0;
+  }
 
-   weight: 100%
-   height: auto 
- *
- *
+  legend {
+    font-size: 38px;
+    color: var(--orange-dark);
+    font-style: normal;
+    font-weight: 500;
+    text-align: left;
+    padding-bottom: 40px;
+  }
 
- __________flex direction: column; for mobile, flex direction row for desktop
- products and product features
- feature
- feature
- feature 
+  input {
+    width: 100%;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    padding: 10px 0 5px 5px;
+    border-color: var(--grey-light);
+    border-radius: 4px;
```

Comment:
problem with the border radius, I add 4px for all sides but top and left side are stronger then bottom and right, I'm not able to find where is this coming from? I did not declare border elsewhere.

Reply:
This is unrelated to the border radius (you can see it's the same even if you remove the border-radius). Remember the border-radius on controls how round the corners are. The coloring of the border is determining the difference between the sides. Try using the `border` property to change that!

You can see examples here: https://www.w3schools.com/cssref/css3_pr_border-radius.asp

Reply:
My apologies, I must been half asleep when I was writing this, I meant border not border-radius. But I can see my mistake, I was relying on inherited border value and then I added color, what I should do is to declare again "border: 1px"  and then add color.